Dockerfile

```diff
@@ -10,9 +10,12 @@ ENV LAST_UPDATE=2016-08-21
 RUN apt-get update && \
     apt-get upgrade -y
 
+# Install tzdata and locales
+RUN apt-get install tzdata locales
+
 # Set the timezone
-RUN echo "Europe/Berlin" | tee /etc/timezone && \
-    ln -fs /usr/share/zoneinfo/Europe/Berlin /etc/localtime && \
+RUN echo "America/Los_Angeles" | tee /etc/timezone && \
```

Comment:
I would suggest using an argument for the timezone value. That will make this dockerfile more reusable by others.